front/src/plugins/vuetify.js

```diff
@@ -4,4 +4,44 @@ import Vuetify from 'vuetify/lib';
 Vue.use(Vuetify);
 
 export default new Vuetify({
+  icons: {
```

Comment:
Alexandr will be sad, theme of vuetify is his task

Reply:
I needed colors for the the user menu (